### DISCUSSION TITLE
Dropdown is overflow from modal

### DISCUSSION THREAD

**Discussion by hajanajubudeen88** [fomantic/Fomantic-UI#695] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
<h1> Bug Report <h1>

<h3>Dropdown is overflow from modal </h3> 

## Steps to reproduce
1. I created a model with text fields and dropdown
2. In dropdown i have 2 values 
3. The first value is "Chicken"
4. The second value is "I like Vegetables I like Vegetables I like Vegetables"
5. when i choose second value i will automatically jumb out from the modal

## Actual result
![test](https://user-images.githubusercontent.com/32840808/56730887-b6ab6b00-6776-11e9-89a7-38bfcda58d0b.PNG)

## Testcase
https://jsfiddle.net/hajanajubudeen88/oarfnz3e/25/

**Comment by ko2in** [comment, marked as answer]
@hajanajubudeen88 , remove `image` from the class name of your modal content. Change 
`<div class="image content">` to `<div class="content">`

[Here](https://jsfiddle.net/7x3z2Lh9/) is the modified version of your fiddle.

**Comment by hajanajubudeen88** [comment]
**@ko2in** Awesome it's working perfectly Thank you
